server/config.js

```diff
@@ -124,14 +143,10 @@ module.exports = {
     licences: {
       taggingCompanyTelephone: get('TAGGING_CO_PHONE', '0800 137 291'),
       pdfOptions: {
-        format: 'A4',
-        margin: {
-          top: '80px',
-          bottom: '70px',
-          left: '50px',
-          right: '30px',
-        },
-        displayHeaderFooter: true,
+        marginTop: '0.8',
+        marginBottom: '0.7',
+        marginLeft: '0.55',
+        marginRight: '0.35',
       },
```

Comment:
Tweaked to be Gotenberg friendly.  A4 is default paper size so not needed.  Margins are in inches.